sycl/source/detail/jit_compiler.cpp

```diff
@@ -825,21 +825,20 @@ jit_compiler::fuseKernels(QueueImplPtr Queue,
   static size_t FusedKernelNameIndex = 0;
   std::stringstream FusedKernelName;
   FusedKernelName << "fused_" << FusedKernelNameIndex++;
-  ::jit_compiler::Config JITConfig;
+  ::jit_compiler::resetConfiguration();
   bool DebugEnabled =
       detail::SYCLConfig<detail::SYCL_RT_WARNING_LEVEL>::get() > 0;
-  JITConfig.set<::jit_compiler::option::JITEnableVerbose>(DebugEnabled);
-  JITConfig.set<::jit_compiler::option::JITEnableCaching>(
+  ::jit_compiler::set<::jit_compiler::option::JITEnableVerbose>(DebugEnabled);
+  ::jit_compiler::set<::jit_compiler::option::JITEnableCaching>(
       detail::SYCLConfig<detail::SYCL_ENABLE_FUSION_CACHING>::get());
 
   ::jit_compiler::TargetInfo TargetInfo = getTargetInfo(Queue);
   ::jit_compiler::BinaryFormat TargetFormat = TargetInfo.getFormat();
-  JITConfig.set<::jit_compiler::option::JITTargetInfo>(TargetInfo);
+  ::jit_compiler::set<::jit_compiler::option::JITTargetInfo>(TargetInfo);
```

Comment:
```suggestion
  ::jit_compiler::set<::jit_compiler::option::JITTargetInfo>(std::move(TargetInfo));
```
Avoid copy (allowed by in-place construction)